Habitica/src/main/java/com/habitrpg/android/habitica/models/inventory/AnimalExtension.kt

```diff
@@ -0,0 +1,23 @@
+package com.habitrpg.android.habitica.models.inventory
+
+import android.content.Context
+import com.habitrpg.android.habitica.R
+
+fun Animal.getTranslatedType(c: Context?): String {
+    var currType = type
+
+    if (currType == "drop") {
+        currType = c?.getString(R.string.standard)
+    }
+    if (currType == "quest") {
+        currType = c?.getString(R.string.quest)
+    }
+    if (currType == "wacky") {
+        currType = c?.getString(R.string.wacky)
+    }
+    if (currType == "special") {
+        currType = c?.getString(R.string.special)
+    }
+
+    return currType
```

Comment:
This looks correct!

One small change: I would prefer it to use one if/else construct instead of multiple if. Reason being that with this, even if it has already found the correct translation it will still check all the following cases. With an if/else, the following if-checks are not evaluated once one is true.
Alternatively this would also be a good case for kotlins [when](https://kotlinlang.org/docs/reference/control-flow.html).